[2024-06-18T11:28:17.020+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:28:17.050+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:26:00+00:00 [queued]>
[2024-06-18T11:28:17.064+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:26:00+00:00 [queued]>
[2024-06-18T11:28:17.064+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-06-18T11:28:17.080+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): skip_run> on 2024-06-18 11:26:00+00:00
[2024-06-18T11:28:17.085+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4836) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:28:17.087+0000] {standard_task_runner.py:63} INFO - Started process 4837 to run task
[2024-06-18T11:28:17.087+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'alerting_skipped', 'skip_run', 'scheduled__2024-06-18T11:26:00+00:00', '--job-id', '703', '--raw', '--subdir', 'DAGS_FOLDER/alerting_skipped.py', '--cfg-path', '/tmp/tmp6ayd1euk']
[2024-06-18T11:28:17.089+0000] {standard_task_runner.py:91} INFO - Job 703: Subtask skip_run
[2024-06-18T11:28:17.132+0000] {task_command.py:426} INFO - Running <TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:26:00+00:00 [running]> on host fa5c5d972e05
[2024-06-18T11:28:17.224+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alerting_skipped' AIRFLOW_CTX_TASK_ID='skip_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T11:26:00+00:00'
[2024-06-18T11:28:17.224+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:28:17.239+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:28:17.240+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/helper_functions.py", line 47, in alert_notif_skipped
    session = Session()
              ^^^^^^^
NameError: name 'Session' is not defined
[2024-06-18T11:28:17.249+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=alerting_skipped, task_id=skip_run, run_id=scheduled__2024-06-18T11:26:00+00:00, execution_date=20240618T112600, start_date=20240618T112817, end_date=20240618T112817
[2024-06-18T11:28:17.261+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 703 for task skip_run (name 'Session' is not defined; 4837)
[2024-06-18T11:28:17.302+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-18T11:28:17.316+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T11:28:17.317+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
